### DISCUSSION TITLE
How to change the View's back button color (in material 2)

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#768] [Q&A] [answered]
I just want to know if this is a bug or a a default stuff when using Material 2.
Running the below code(gotten from the docs), the color of the back button is transparent when I modify the `bgcolor` and `textcolor` of my `appbar`:

```python
import flet as ft


def main(page: ft.Page):
    page.title = "Routes Example"
    page.theme_mode = "light"
    page.theme = ft.Theme(
        use_material3=False,  # use material 2 to test this issue
    )

    def route_change(route):
        page.views.clear()
        page.views.append(
            ft.View(
                "/",
                [
                    ft.AppBar(title=ft.Text("Flet app", color="black"), bgcolor=ft.colors.WHITE),
                    ft.ElevatedButton("Visit Store", on_click=lambda _: page.go("/store")),
                ],
            )
        )
        if page.route == "/store":
            page.views.append(
                ft.View(
                    "/store",
                    [
                        ft.AppBar(title=ft.Text("Store", color="black"), bgcolor=ft.colors.WHITE),
                        ft.ElevatedButton("Go Home", on_click=lambda _: page.go("/")),
                    ],
                )
            )
        page.update()

    def view_pop(view):
        page.views.pop()
        top_view = page.views[-1]
        page.go(top_view.route)

    page.on_route_change = route_change
    page.on_view_pop = view_pop
    page.go(page.route)


ft.app(target=main)
```

Result:
![bug](https://user-images.githubusercontent.com/98978078/209579575-805d8286-3f21-4055-9e83-f4d236bb2e90.png)

When in Material 3(instead of 2), this issue goes away though. (but I need the m2 for a project using views --)

**Comment by JorgeAlvarez21** [comment, marked as answer]
The button in Material three is white by default, to make it black you can pass the parameter to `ft.Appbar(color=ft.colors.BLACK)`

**Reply by ndonkoHenri** [reply]
Thank you very much! I completely missed the `color` property.

**Reply by JorgeAlvarez21** [reply]
Np, glad to be of help! Btw I have used some of your Flet utility tools several times, they're awesome!

**Reply by ndonkoHenri** [reply]
Oh. Thanks.
I will be happy to have more feedbacks(ex: issues, suggestions)

Anyway, I am planning a big update to that project. I haven't yet started though, but it's already planned.
Stay tuned, and thanks again for your appreciation/return. 😉
